Documentation/Blazorise.Docs/Pages/Docs/Extensions/DataGrid/Examples/DataGridColumnChooserExample.razor

```diff
@@ -0,0 +1,47 @@
+﻿@namespace Blazorise.Docs.Docs.Examples
+
+<DataGrid TItem="Employee"
+          Data="inMemoryData"
+          Responsive
+          ShowColumnChooser
+          PagerPosition="DataGridPagerPosition.Top"
+          ShowPager
+          ShowPageSizes>
+    <DataGridColumns>
```

Comment:
What do you think we simplify Examples by not defining the columns manually? We now have autogenerate columns from the Model.

Reply:
I like it alot for simplifying the examples.
What I get a little bit worried is not showing how columns get built on the examples.

Reply:
Well, doesn't need to on all examples. Just some that we want to focus on specific feature. Like this column chooser. having too many columns in the code samples add a noise IMO.

Reply:
Right, but most examples, **DO** focus on a specific feature. So they'll be missing the columns. 
You know how less experienced users are sometimes... 
Anyways can't hurt trying. Do you think we should only apply to this one, or apply to all we think its worth it?